run_tests.sh

```diff
@@ -246,6 +246,7 @@ TOOL_SHED_TEST_TMP_DIR          Defaults to random /tmp directory - place for
                                 tool shed test server files to be placed.
 TOOL_SHED_TEST_OMIT_GALAXY      Do not launch a Galaxy server for tool shed
                                 testing.
+GALAXY_TEST_UVICORN_ACCESS_LOG  Do not output uvicorn access messages to log if non-enmpty
```

Comment:
```suggestion
GALAXY_TEST_DISABLE_ACCESS_LOG  Do not log access messages
```
I think we don't need to be that specific here. We won't disable paster or uwsgi access logs, but those aren't used by default. Also by naming it that way we don't need to parse the value of `GALAXY_TEST_DISABLE_ACCESS_LOG`.